### DISCUSSION TITLE
`useController` TEXTFIELD documentation

### DISCUSSION THREAD

**Discussion by ehabsuleiman** [react-hook-form/react-hook-form#8149] [Q&A] [answered]
## useController TextField api documentaion
in the documentation, `control` is a prop and its also being redefined using the  useForm() hok, i'm wondering why is that the case here, shouldn't it only be passed in as prop here? since the goal here is to make a reusable Controlled input.
``` js
function Input({ control, name }) {
  const { control } = useForm();
```

### Image
![image](https://user-images.githubusercontent.com/32402313/162282737-64948c54-478f-40fc-9abb-24e6a244de74.png)

**Comment by Moshyfawn** [comment, marked as answer]
You're right. There shouldn't be any `useform` calls in this example. Will remove it

**Reply by bluebill1049** [reply]
done. :)
